JavaScript/index.d.ts

```diff
@@ -1,1455 +1,3 @@
-/**
```

Comment:
why this file is removed?

Reply:
Same reason as the last comment, will fix in new PR's